src/pages/workspace/withPolicyConnections.tsx

```diff
@@ -39,7 +41,7 @@ function withPolicyConnections<TProps extends WithPolicyConnectionsProps>(Wrappe
         const prevHasConnectionsDataBeenFetched = usePrevious(hasConnectionsDataBeenFetched);
 
         useEffect(() => {
-            if (prevHasConnectionsDataBeenFetched !== undefined || !isBoolean(hasConnectionsDataBeenFetched)) {
+            if (!isBoolean(hasConnectionsDataBeenFetched) || prevHasConnectionsDataBeenFetched === hasConnectionsDataBeenFetched) {
```

Comment:
Why are we changing the condition here? We’re not touching `POLICY_HAS_CONNECTIONS_DATA_BEEN_FETCHED`, so are you trying to fix an existing issue, or do our changes require this update?